docs/sigmod2021.md

```diff
@@ -0,0 +1,37 @@
+# ACM SIGMOD 2021
+
+Welcome participants of the [ACM SIGMOD 2021 programming contest](https://dbgroup.ing.unimore.it/sigmod21contest/)! This page contains some special information for you to get
+started more easily with our tool.
+
+Feel free to open an [issue](https://github.com/HPI-Information-Systems/snowman/issues/new/choose) in case you find a bug or see space for improvement :)
+
+## Introduction
+
+This tool will help you to compare and evaluate your data matching solutions. You can upload experiment results from
+your data matching solution and then compare it with a goldstandard, compare two experiment runs with each other or
+calculate binary metrics like precision or recall.
+
+We are working closely with the contest team to allow for a seamless experience.
+
+## Setup
+
+To use snowman, you'll have to download the latest [release (Github Releases)](https://github.com/HPI-Information-Systems/snowman/releases) of
+snowman for your specific os. After that you are able to start snowman!  
+As a first step, we'd suggest for you to start with our section [Basic Usage](/basic_usage/introduction)
+for a guide of how to benchmark and evaluate with snowman.
```

Comment:
evaluate what?

Reply:
Well - the algorithm the user is working with ;) Will change this :)